### PR TITLE
Set version artificially for the test template runs

### PR DESCRIPTION
**Pull request by chidozieononiwu**
This allows template (Test) run to go all the way to release with artificially generated versions.
Sample run https://dev.azure.com/azure-sdk/internal/_build/results?buildId=559246&view=results

### CODE REVIEW

File: eng/scripts/SetTestPipelineVersion.ps1 [hunk 1]
```diff
@@ -0,0 +1,31 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
+# This is to help with testing the release pipeline.
+. "${PSScriptRoot}\..\common\scripts\common.ps1"
+$latestTags = git tag -l "azure-sdk-template*beta*"
```
Comment (weshaggard) [inline eng/scripts/SetTestPipelineVersion.ps1:4]: Why `*beta*` as opposed to just `_*`?

File: eng/scripts/SetTestPipelineVersion.ps1 [hunk 2]
```diff
@@ -0,0 +1,31 @@
+# Overides the project file and CHANGELOG.md for the template project using the next publishable version
```
Comment (weshaggard) [inline eng/scripts/SetTestPipelineVersion.ps1:1]: This script is called SetTestePipelineVersion.ps1 but it really is specific to the template package. Not a big deal but it seems a little miss-leading. When you come back around to clean this up a little we should consider a way to share the common parts of this in eng/common and then call the language specific version incrementing and changelog update.

**Review by weshaggard — APPROVED** [review]

File: eng/scripts/SetTestPipelineVersion.ps1 (hunk 1 above)
Comment (chidozieononiwu) [reply eng/scripts/SetTestPipelineVersion.ps1:4]: Am trying to not have to filter sort with the preview version. I don't assume we need to publish a GA version from the template pipeline.

**Pull request merged**